app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self. hidden = hidden
```

Comment:
There's an extra space in `self. hidden` which should be corrected to `self.hidden`.